projects/packages/forms/src/blocks/input-image-option/default-attributes.ts

```diff
@@ -0,0 +1,19 @@
+export default {
+	style: {
+		border: {
+			width: '1px',
+			color: '#dee2e6',
+			radius: '4px',
+			style: 'solid',
+		},
+		spacing: {
+			margin: '0',
+			padding: '8px',
+		},
+		color: {
+			background: '#f8f9fa',
+			text: '#212529',
+		},
+	},
+	fontSize: 'medium',
+};
```

Comment:
Is there a away we could inherit theme.json values for some of these things, especially colours but also border radius? That way they would blend into the theme and rest of the form inputs automatically rather than be fixed style from the start.